ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -477,14 +567,46 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         volumePanelRef.current.removeEventListener('wheel', volumePanelScrollHandlerRef.current);
       }
 
+      const chapterMarkers = document.getElementsByClassName('vjs-chapter-marker');
+      while (chapterMarkers.length > 0) {
+        // $FlowIssue
+        chapterMarkers[0].parentNode?.removeChild(chapterMarkers[0]);
+      }
```

Comment:
_--- Can ship first so as to not invalidate testing, but consider in near future ---_

Can we try moving this into `chapters.jsx :: load` and perform the action inside a `player.on('playerClosed')`?

I have some pending tweaks to this area, and this will be more intuitive framework-wise, plus we don't need to hunt various places to debug later.